src/main/kotlin/blackjack/domain/PlayerBettings.kt

```diff
@@ -0,0 +1,27 @@
+package blackjack.domain
+
+@JvmInline
+value class PlayerBettings private constructor(val bettingsMap: Map<Name, Money>) {
+
+    private fun getMoney(name: Name) = bettingsMap[name]
+        ?: throw NullPointerException(BETTINGS_MAP_NULL_POINTER_EXCEPTION_MESSAGE)
+
+    fun calculateDealerProfit(dealer: Dealer, players: Players): Int {
+        return players.players
+            .map { dealer.profit(it, getMoney(it.name)) }
+            .reduce { acc: Int, profit: Int -> acc + profit }
```

Comment:
```suggestion
        return players.players
            .filter { bettingsMap.containsKey(it.name) }
            .map { dealer.profit(it, bettingsMap[it.name]!!) }
            .reduce { acc: Int, profit: Int -> acc + profit }
```
filter를 통해서 key가 있는 map만 전달하게 한다면, !!을 활용해도 좋을 것 같아요

Reply:
null이 아니라는 보장이 있다면 !!를 써도 괜찮긴 하겠네요! 적용했습니다!